templates/.storybook/config.js

```diff
@@ -1 +1,24 @@
-import '@stoplight/scripts/.storybook/config';
+import { addDecorator, configure } from '@storybook/react';
```

Comment:
What's changed here:
* better support for themes
* removed configure call - in some cases we had to copy the entire config just because stories were located somewhere else. Happy to revert that change if you think it's worth to have that call out of the box.